### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by ccremer**
#### What this PR does / why we need it:

Adds a file to define Code owners

As Code owner, you are invited to review Pull requests, both from contributors as well as Bots like Renovate.
Reviewing a PR can also mean "Close, won't do", they don't have to be merged, but it's about doing conscious decisions

#### Checklist

- [x] PR contains the label that identifies the type of change, which is one of
      [ `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency` ]

### CODE REVIEW

**Comment by ccremer** [comment]
NOTE: I have added maintainers to charts where I found out the contributors from past pull requests.
If you have any objections, let me know or suggest a correction.

**Comment by ccremer** [comment]
NOTE2: Not all charts have maintainers yet. I'm not yet sure what to do with those.

**Review by tobru — APPROVED** [review]

**Comment by ZuGao** [comment]
You can add me to trifid as well

**Review by mhutter — APPROVED** [review]

**Review by corvus-ch — APPROVED** [review]
Some of the charts are missing. Is this on purpose?

* haproxy
* metrics-server
* openshift-oauth-proxy
* prometheus-blackbox-exporter
* snappass

**Comment by ccremer** [comment]
> Some of the charts are missing. Is this on purpose?
> 
>     * haproxy
> 
>     * metrics-server
> 
>     * openshift-oauth-proxy
> 
>     * prometheus-blackbox-exporter
> 
>     * snappass

See my comment NOTE2...

**Review by psy-q — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# As a Code owner, you are requested for Pull Request reviews
+
+# Used by VSHN as part of product offering
+/appuio/k8up              @appuio/tarazed
+/appuio/mariadb-galera    @appuio/tarazed
+/appuio/redis             @appuio/tarazed
+/appuio/generic           @appuio/tarazed
+/appuio/openshift-route   @appuio/tarazed
+/appuio/secret            @appuio/tarazed
```
Comment (anothertobi) [inline CODEOWNERS:11]:
```suggestion
/appuio/secret            @appuio/tarazed
/appuio/haproxy           @appuio/tarazed @anothertobi
```

We use the `/appuio/haproxy` chart in the same product offering as the `redis` and `mariadb-galera` charts. IIRC, I was involved in the initial implementation of it, so you can also add me besides `@appuio/tarazed`.

Comment (ccremer) [reply CODEOWNERS:11]: Thanks, will do. Would you also be interested as owner for mariadb-galera and redis as well?

Comment (anothertobi) [reply CODEOWNERS:11]: As I don't know them very well, that doesn't make sense. The HAProxy chart existed >1.5y before the other two got forked.

**Comment by ccremer** [comment]
Thanks for the reviews!
Going to merge now.

**Pull request merged**